File: C&C++/Arshdeep Kaur/Day5_2.cpp
//Write a program to find the LCM (Least Common Multiple) of two numbers using a while loop.

#include <iostream>
using namespace std;

int main() {
	int num1, num2;

	cout << "Enter the two numbers to find their LCM: ";
	cin >> num1 >> num2;

	int max = (num1 > num2) ? num1 : num2;

	while (1) {
		if (max % num1 == 0 && max % num2 == 0) {
			cout << "LCM of " << num1 << " and " << num2 << " is: " << max;
			break;
		}
		++max;
	}

	return 0;
}

File: Python/Arshdeep Kaur/Day5_2.py
#Write a Python program that prints all the prime numbers between two given numbers provided by the user.

num1 = int(input("Enter the lower range number: "))
num2 = int(input("Enter the upper range number: "))

#assuming that both the numbers are inclusive
print("The prime numbers between", num1, "and", num2, "are: ")

for i in range(num1, num2+1):
    if i > 1:
        for j in range(2, i):
            if (i % j == 0):
                break
        else:
            print(i)
